addons/mrp/data/mrp_demo.xml

```diff
@@ -427,6 +427,7 @@
         </record>
 
          <record id="product_product_table_kit_product_template" model="product.template">
+            <field name="type">consu</field>
```

Comment:
You don't need to precise the product's type for both the template and variant, the change on line 420 is enough (and as `consu` is the default value for the product type, you could even delete this line instead of update `product` to `consu`, but I think it's safer to precise we want a consumable for this product like you did :+1: )